resolver-online-offline.mdx

```diff
@@ -27,7 +27,7 @@ This timeseries data is highly compressible,
 as many features change less frequently than they are sampled
 (for example, users on your platform might very rarely change their names.)
 Chalk uses the column store
-[Timescale](https://timescale.com) or the data warehouse [BigQuery](https://cloud.google.com/bigquery)for this data.
+[Timescale](https://timescale.com) or the data warehouse [BigQuery](https://cloud.google.com/bigquery) for this data.
```

Comment:
I know that this is not an addition that you made to the docs but this whole paragraph hasn't been true for a long time - we don't use timescale for any production offline stores, timescale doesn't really use columnar storage, and our timescale implementation is really just a postgres implementation. It would be amazing if we could update this paragraph to describe our multi-db support for different offline stores in a more accurate way. Something like:

(drop the "timeseries data" bit) "Chalk uses different data warehouse systems for large-scale offline storage based on customer needs and deployment type, like BigQuery, Snowflake, redshift, etc."